### PR TITLE
Baseline

### PR DESCRIPTION
**Pull request by RuochenZhao**

### CODE REVIEW

File: EDA/KDWD_EDA.ipynb [hunk 1]
```diff
@@ -1665,7 +1665,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.7.4"
```
Comment (johannes-kk) [inline EDA/KDWD_EDA.ipynb:1668]: Was this Python version downgrade intentional? I expect this would break a lot of dependencies

**Comment by johannes-kk** [comment]
The build test failed on black (i.e. code formatting). Did you set up the pre-commit hooks in the `README.md`?

File: EDA/KDWD_EDA.ipynb (hunk 1 above)
Comment (RuochenZhao) [reply EDA/KDWD_EDA.ipynb:1668]: This was Hardik's notebook. But it isn't intentional I think. Both versions would work. There aren't many dependencies on our repo right now.

**Comment by johannes-kk** [comment]
FYI @RuochenZhao I changed `black` style and `isort` import sorting checks to not just complain about it if something is off, but instead attempt to fix it automatically and approve the PR if that worked.

**Pull request merged**